packages/http.go

```diff
@@ -63,7 +63,7 @@ func ServePackageSignature(logger *zap.Logger, w http.ResponseWriter, r *http.Re
 }
 
 func serveLocalPackage(logger *zap.Logger, w http.ResponseWriter, r *http.Request, p *Package, packagePath string) {
-	span, _ := apm.StartSpan(r.Context(), "ServePackage", "app")
+	span, _ := apm.StartSpan(r.Context(), "ServeLocalPackage", "app")
```

Comment:
There was already an Span called `ServePackage` at line 31.

Reply:
If it is not changed, it appears twice the same Span name:

![image](https://github.com/user-attachments/assets/853a4f7e-ebc3-4523-aa60-de4a6f8c1cfd)

I'd say it would be better to have different names there so it is more specific.